### PR TITLE
Bug 1465355 - Fix Treeherder forgetting being logged in

### PR DESCRIPTION
**Pull request by camd**

### CODE REVIEW

File: ui/models/user.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ export default class UserModel {
   }
 
   static get() {
-    return fetch(`${uri}`)
+    // TODO: This credentials value can be removed in July once Firefox 62 ships and it is the default.
+    return fetch(`${uri}`, { credentials: "same-origin" })
```
Comment (edmorley) [inline ui/models/user.js]: I think the methods in `ui/helpers/http.js` will be affected too.

Comment (camd) [reply ui/models/user.js]: Crap.  You're right, of course.  Thanks!

File: ui/helpers/http.js [hunk 2]
```diff
@@ -8,11 +8,13 @@ const generateHeaders = function generateHeaders() {
   });
 };
 
+// TODO: The credentials param can be removed in July once Firefox 62 ships and it is the default.
 export const create = function postJson(uri, data) {
   return fetch(uri, {
     method: 'POST',
     headers: generateHeaders(),
     body: JSON.stringify(data),
+    credentials: "same-origin",
```
Comment (edmorley) [inline ui/helpers/http.js]: single quotes in JS (double in JSX attributes)

(the quotes rule is still turned off in the linter - I'll see about getting it enabled soon)

**Comment by camd** [comment]
ahh, right.  Dang, I STILL default to double-quotes in my typing for some reason.  It's a hard habit to break for some reason...

**Pull request merged**